wear/build.gradle.kts

```diff
@@ -93,6 +93,7 @@ dependencies {
     implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.6.21")
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.4")
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-guava:1.6.4")
+    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-play-services:1.6.4")
```

Comment:
In the latest commit you removed the use of `await()` for the phone listener task, which means this dependency is now unused. Why was it removed / shouldn't this dependency be removed as well now?

Reply:
It was removed since I didn't see a fullImplementation equivalent in this module (although I have no idea if a no-closed-source app is possible on Wear OS). I forgot to remove it here

Reply:
There is no equivalent for the Wear module, as Wear without Google services doesn't exist. In other words, no need to remove it for Wear.